src/server/api/routers/auth.ts

```
@@ -104,4 +108,132 @@ export const authRouter = createTRPCRouter({
+
+      return { token, userId: user.id };
+    }),
+
+  requestMobileOtp: publicProcedure
+    .input(z.object({ email: z.string().email() }))
+    .mutation(async ({ input }) => {
+      const normalizedEmail = input.email.toLowerCase();
+
+      const user = await db.user.findUnique({
+        where: { email: normalizedEmail },
+        select: { id: true },
```

ESLint

[error] Expected object keys to be in natural insensitive ascending order. 'select' should be before 'where'.

(sort-keys-fix/sort-keys-fix)